homebrew/br/index.html

```diff
@@ -2,28 +2,25 @@
 <html>
 <head>
     <meta charset="utf-8"/>
-	<meta name="viewport" content="width=640, initial-scale=1, user-scalable=yes"/>
+    <meta name="viewport" content="width=640, initial-scale=1, user-scalable=yes"/>
     <title>3DS Homebrew | The Homebrew Database</title>
-    <link rel="shortcut icon" href="../../favicon.ico" type="image/x-icon"/>
-    <link rel="stylesheet" href="../../main.css"/>
+    <base href="/3ds/">
```

Comment:
Totally forgot about the base tag lol
Mainly because I was using a local webserver for testing.